rasterio/rio/info.py

```diff
@@ -38,6 +128,13 @@ def edit(ctx, input, nodata, crs, transform, tags):
     or JSON-encoded GDAL geotransform arrays like
 
       [101985.0, 300.038, 0.0, 2826915.0, 0.0, -300.042]
+
+    Metadata items may also be read from an existing dataset using a
+    combination of the --like, --crs, --nodata, --transform, and --all
+    options.
+
+      rio-edit-info example.tif --all --like template.tif
```

Comment:
@sgillies typo: ```rio-edit-info``` -> ```rio edit-info```

Also, the description above would be more clear like this:

```
    Metadata items may also be read from an existing dataset using a
    combination of the --like option with at least one of --all, --crs, 
    --nodata, and --transform options.
```

It is unclear what values are going to be copied from template when ```--like``` is provided but others are not.  When I tested this option only, I got several unexpected results given that ```--all``` is supposed to be ```False``` by default:

```
$ cp tests/data/shade.tif /tmp/test.tif

$ rio info /tmp/test.tif
{"count": 1, "crs": "EPSG:3857", "res": [9.5546285343, 9.5546285343], "bounds": [-11858134.818413004, 4803914.3530251775, -11848350.87879388, 4813698.2926443005], "dtype": "uint8", "driver": "GTiff", "transform": [9.5546285343, 0.0, -11858134.818413004, 0.0, -9.5546285343, 4813698.2926443005], "lnglat": [-106.47949217280885, 39.605688173875606], "height": 1024, "width": 1024, "shape": [1024, 1024], "blockxsize": 1024, "tiled": false, "blockysize": 8, "nodata": 255.0}

$ rio edit-info /tmp/test.tif --like tests/data/RGB.byte.tif

$ rio info /tmp/test.tif
{"count": 1, "crs": "EPSG:32618", "res": [300.0379266750948, 300.041782729805], "bounds": [101985.0, 2519672.2144846795, 409223.8369152971, 2826915.0], "dtype": "uint8", "driver": "GTiff", "transform": [300.0379266750948, 0.0, 101985.0, 0.0, -300.041782729805, 2826915.0], "lnglat": [-77.40522561315726, 24.153251565772365], "height": 1024, "width": 1024, "shape": [1024, 1024], "blockxsize": 1024, "tiled": false, "blockysize": 8, "nodata": 255.0}
```

Looks like crs and transform were copied across but not nodata.